docs/variables/user.md

```diff
@@ -4,137 +4,410 @@ id: user
 
 # $(user)
 
-This variable returns either the first argument in the command, or if that is unavailable, the username of the person who triggered the command, or a custom specified user (for example `$(user aiden)` would use `aiden` as the user).
+Returns a range of metadata around a (specified) user's channel that can be accessed via multiple different variable members for each unique piece of information.
 
-:::caution Timers are not supported.
+:::caution Timers are not supported!
 
-The variable will not return any value in a timer.
+Timers are executed on an interval by Fossabot itself and therefore this variable cannot be used inside of them.
```

Comment:
It should be supported if it's used with a parameter i think? like `$(user.uptime aiden)` etc

Reply:
I'll make sure to check again and will fix if you're right.